docs/samples/Microsoft.ML.Samples/Dynamic/LdaTransform.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class LdaTransformExample
+    {
+        public static void LdaTransform()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleTopicsData> data = SamplesUtils.DatasetUtils.GetTopicsData();
+            var trainData = ml.CreateStreamingDataView(data);
+
+            // Preview of one of the columns of the the topics data. 
+            // The Review column contains the keys associated with a particular body of text.  
+            //
+            // Review                               
+            // "animals birds cats dogs fish horse" 
+            // "horse birds house fish duck cats"   
+            // "car truck driver bus pickup"       
+            // "car truck driver bus pickup horse"
+
+            // A pipeline for featurizing the "Review" column
+            string ldaFeatures = "LdaFeatures";
+            var pipeline = ml.Transforms.Text.ProduceWordBags("Review").
+                Append(ml.Transforms.Text.LatentDirichletAllocation("Review", ldaFeatures, numTopic:3));
```

Comment:
>LatentDirichletAllocation [](start = 42, length = 25)

just asking: besides numTopic, is there other params that the user might tune often?

Reply:
i doubt users tune other parameters besides numTopic

---
In reply to: [237660759](https://github.com/dotnet/machinelearning/pull/1782#discussion_r237660759) [](ancestors = 237660759)